SCClassLibrary/Common/GUI/Base/Grid.sc

```diff
@@ -2,6 +2,7 @@ DrawGrid {
 
 	var <bounds, <>x, <>y;
 	var <>opacity=0.7, <>smoothing=false, <>linePattern;
+	var <>testView;
```

Comment:
is there a reason this is also a setter?

Reply:
Yes, in the *test method we don't have access to the testView as a variable to set it, so it needs to be accessed as <instance>.testView

Reply:
> Yes, in the *test method we don't have access to the testView as a variable to set it, so it needs to be accessed as `<instance>.testView_`

Reply:
I ended up removing the changes associated with `*test`, per our DM discussion.